frontend/ui/widget/keyvaluepage.lua

```diff
@@ -337,6 +337,7 @@ local KeyValuePage = InputContainer:new{
     -- aligment of value when key or value overflows its reserved width (for
     -- now: 50%): "left" (stick to key), "right" (stick to scren right border)
     value_overflow_align = "left",
+    -- single_page = nil,
```

Comment:
Leave it uncommented (like we have above for width/height. Having it commented out, we may think it's an old option that is no longer valid.